README.md

```diff
@@ -12,8 +12,8 @@
 - Initialise and build the VE sub-module
 - `cd lib/ve`
 - `npm install`
-- Download, install and run [Apertium-APy](https://github.com/apertium/apertium-apy) .
- (or alternatively, edit `demo/demo.ll.js` and change `http://localhost:2737/translate` to point to a public Apertium-APy instance)
+- `cp environment.json.apertium-example environment.json`
+ (or alternatively, use `environment.json.yandex-example` and put in your Yandex API key)
 - Run a local static webserver in the repository directory: `python3 -m http.server`
```

Comment:
we should probably be less specific here about which web server to use, we shouldn't suggest python3 is a dependency

Reply:
OK sure, but my patch doesn't touch that line at all